policy-controller/k8s/status/src/index.rs

```diff
@@ -21,19 +24,23 @@ pub const STATUS_CONTROLLER_NAME: &str = "status-controller";
 pub type SharedIndex = Arc<RwLock<Index>>;
 
 pub struct Controller {
+    claims: Receiver<Arc<Claim>>,
     client: k8s::Client,
+    name: String,
     updates: UnboundedReceiver<Update>,
+
+    // True if this status controller is the leader — false otherwise.
```

Comment:
tiny nit, definitely not actually important: prefer doc comments for struct fields even when the struct is not public, as they can be rendered by `cargo doc --document-private-items`
```suggestion
    /// True if this status controller is the leader — false otherwise.
```